rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -95,13 +199,7 @@ docker run --name $docker_name \
   --label PAI_JOB_VC_NAME={{{ jobData.virtualCluster }}} \
   --label PAI_USER_NAME={{{ jobData.userName }}} \
   --label PAI_CURRENT_TASK_ROLE_NAME={{{ taskData.name }}} \
-  --env APP_ID=$APP_ID \
-  --env FRAMEWORK_NAME=$FRAMEWORK_NAME \
-  --env HADOOP_USER_NAME=$HADOOP_USER_NAME \
-  --env PAI_TASK_INDEX=$TASK_INDEX \
-  --env PAI_CONTAINER_HOST_IP=$CONTAINER_IP \
-  --env PAI_CONTAINER_HOST_PORT_LIST=$CONTAINER_PORTS \
-  --env PAI_CONTAINER_ID=$CONTAINER_ID \
+  --env-file $ENV_LIST \
   --entrypoint '/bin/bash' {{{ jobData.image }}} \
   '-c' 'hdfs dfs -get {{{ hdfsUri }}}/Container/$HADOOP_USER_NAME/$FRAMEWORK_NAME/DockerContainerScripts/{{{ idx }}}.sh bootstrap.sh && /bin/bash bootstrap.sh' \
```

Comment:
>hdfs dfs -get {{{ hdfsUri }}}/Container/$HADOOP_USER_NAME/$FRAMEWORK_NAME/DockerContainerScripts/{{{ idx }}}.sh bootstrap.sh [](start = 8, length = 124)

we can do this in yarnContainerScript. and cp to a folder mounted by the dockerContainer.

Reply:
will refine in another PR